dev_docs/contributing/third_party_dependencies.mdx

```diff
@@ -67,6 +67,57 @@ on Github:
   [Permitted Open Source Licenses list](https://github.com/elastic/open-source/blob/main/elastic-product-policy.md#permitted-licenses-list).
 </DocCallOut>
 
+### Dependency evaluation
+
+1. **Snyk health check**
+    - Ensure a minimum health score is >= 80 in [Snyk Advisor](https://snyk.io/advisor).
```

Comment:
Can we explain why we chose a score of `80`, as opposed to any other number? We will be questioned the first time we block a dependency based on a score, so I want to make sure this isn't arbitrary.